compressors/cleancss.py

```diff
@@ -0,0 +1,5 @@
+from pipeline.compressors import SubProcessCompressor
+
+class CleanCSSCompressor(SubProcessCompressor):
+    def compress_css(self, css):
+        return self.execute_command('cleancss', css)
```

Comment:
Mostly organization, yeah. I'm thinking back to that email thread from a while back about repository organization and I'm not sure where something like this fits in.